aggregator/src/aggregator.rs

```diff
@@ -235,14 +237,18 @@ impl<C: Clock> Aggregator<C> {
 
         let upload_decrypt_failure_counter = meter
             .u64_counter("janus_upload_decrypt_failures")
-            .with_description("Number of decryption failures in the /upload endpoint.")
+            .with_description(
+                "Number of decryption failures in the tasks/{task-id}/reports endpoint.",
+            )
```

Comment:
Will changing the description of Prometheus time series affect aggregation of the metric?

Reply:
No, I don't believe so. I think it just shows up in tooltips, etc.